### DISCUSSION TITLE
Exploring adding Block Variations for display WooCommerce Brands Plugin Shortcodes

### DISCUSSION THREAD

**Discussion by ashleyshaw** [lightspeeddevelopment/lsx-design#186] [Ideas]
#### Introduction:
The WooCommerce Brands plugin currently supports shortcode output for brand lists and thumbnails. We’re exploring the idea of creating block-based variations for a more modern WordPress editing experience, allowing users to drag-and-drop brand-related content easily.

#### Problem/Opportunity:
With the rise of block-based themes and editing, shortcodes are becoming less user-friendly. Users would benefit from native blocks that can be styled and positioned in a more intuitive manner.

#### Proposed Solution:
Develop new block variations for the WooCommerce Brands plugin that include features like brand lists, thumbnails, and product collections. These blocks will align with the LSX Design system and offer more flexibility to users in the WordPress block editor.

#### Alternatives Considered:
Continuing to use shortcodes is possible, but block-based editing offers a much more intuitive, modern approach.

#### Challenges:
- Ensuring the blocks are fully compatible with existing WooCommerce features.
- Offering sufficient customization options without overwhelming the user.

#### Call for Feedback:
I’d like to hear opinions on whether the community would find value in block-based variations for brands. What additional features should be included in these blocks?

#### Next Steps:
I’ll start by prototyping a block for brand thumbnails and gather feedback before expanding to other types.

- @ZaredRogers this relates to this issue #184 
- @krugazul where would you register the variations? Do we need a plugin or could we register them using block json into a theme? Registering a block variation might contravene the WordPress Theme Review guidelines.

**Comment by ZaredRogers** [comment]
### Design Update

I have created the following design ideas for this:

#### Single Brand Landing page

![Single Brand Landing](https://github.com/user-attachments/assets/2e6ac141-6718-4a62-a8db-c431f6a0bfe7)


#### Single Branded Product (Banner)

![Single Branded Product - Banner](https://github.com/user-attachments/assets/5e34ca44-3df2-486f-9b0c-9b77db9e86a1)


#### Single Branded Product (Icon)

![Single Branded Product - Icon](https://github.com/user-attachments/assets/aae2b2ca-bb7d-4c4b-8590-6fadf3879133)


#### Single Branded Product (Logo)

![Single Branded Product - Logo](https://github.com/user-attachments/assets/529a06e8-13c1-4c6f-aac1-aba38dc7a38e)


#### Brand Archive (Banner)

![Brand Archive - Banner](https://github.com/user-attachments/assets/8355f747-7b00-433f-906f-1987629d8eb1)


#### Brand Archive (Logo)

![Brand Archive - Logo](https://github.com/user-attachments/assets/f536048a-cd70-4e33-a7d3-0171b63521aa)


#### Brands Landing
![Brands Landing](https://github.com/user-attachments/assets/d81358bd-8d4d-4776-bd50-5f1213644097)
